test-automation-school/src/test/java/test/automation/school/HWTest.java

```diff
@@ -0,0 +1,87 @@
+package test.automation.school;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import test.automation.school.pageobjects.*;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by Max on 5/17/2017.
+ */
+public class HWTest extends BaseTest {
+    HomePage homePage;
+    SearchPage searchPage;
+    SearchResultsPage searchResultsPage;
+
```

Comment:
So in this test class you`ve got 2 tests which are testing search functionality, therefore the main test activities are starting when user goes to Search Page. I would recommend to move HomePage initialization and "switching lang to ru" actions to a setup method, since they are repeating in both tests. However, if one of the goals of the test include to test particularly russian language, then it should be part of the test itself.